sdk/core/corehttp/corehttp/streaming/decoders.py

```diff
@@ -82,7 +82,6 @@ class JSONLDecoder:
 
     def __init__(self) -> None:
         self._data: str = ""
-        self._decoder = codecs.getincrementaldecoder("utf-8")()
         self._line_separators = re.compile(r"\r\n|\n")
```

Comment:
Doing a regex for a simple search is probably overkill. I'd use `str.splitlines()` after checking if there was a trailing universal newline to determine if the last split line should be returned or continued to build on...